### PR TITLE
Added  elementwise version of => to prec-pair

### PR DESCRIPTION
**Pull request by MasonProtter**
#27446

### CODE REVIEW

**Review by JeffBezanson — APPROVED** [review]

**Comment by JeffBezanson** [comment]
Thanks @MasonProtter, and welcome!

I see you're working on porting scmutils to julia!? That's awesome. I took that class and this is something I've wanted for a while :)

**Comment by MasonProtter** [comment]
> Thanks @MasonProtter, and welcome!
Thanks! 

> I see you're working on porting scmutils to julia!? That's awesome. I took that class and this is something I've wanted for a while :)

Yes! I haven't had any time for it in a while but I hope to get back on track soon! SICM is an awesome book and I'm hoping to learn a lot from porting scmutils. I'd like to eventually build a proper, julia native CAS with the lessons from scmutils.

**Comment by MasonProtter** [comment]
Are those failed builds likely problems with the CI tools or could there actually be a problem with such a minor change?

**Comment by KristofferC** [comment]
You can go into the CI logs and see what made them fail and if it is relevant. CI is not rock solid right now.

**Comment by JeffBezanson** [comment]
The test case from #27446 could be added to test/broadcast.jl.

**Comment by StefanKarpinski** [comment]
Looks like you've got whitespace issues.

**Comment by MasonProtter** [comment]
Trying to fix now.

File: test/broadcast.jl [hunk 1]
```diff
@@ -727,6 +727,13 @@ let f(args...) = *(args...)
     @test f.(x..., f.(x..., y, z...), y, z...) == broadcast(f, x..., broadcast(f, x..., y, z...), y, z...) == 120*120
 end
 
+# Issue #27446: Broadcasting pair operator
+let
```
Comment (ararslan) [inline test/broadcast.jl]: Your trailing whitespace is on this line

**Comment by ararslan** [comment]
It would be great if you could add a NEWS entry for this change as well.

**Comment by MasonProtter** [comment]
What section of NEWS.md would be appropriate for this?

**Comment by ararslan** [comment]
I'd put it at the end of "new language features."

File: NEWS.md [hunk 2]
```diff
@@ -14,7 +14,28 @@ New language features
   * Named tuples, with the syntax `(a=1, b=2)`. These behave very similarly to tuples,
     except components can also be accessed by name using dot syntax `t.a` ([#22194]).
 
-  * Keyword argument containers (`kw` in `f(; kw...)`) are now named tuples. Dictionary
+  * Keyw
```
[the remark this answers is not in the document]
Comment (ararslan) [reply NEWS.md]: No worries. Though you can view the diff yourself locally before pushing with `git diff origin/master`. That way you can spot oddities before they show up in the PR.

[the remark this answers is not in the document]
Comment (ararslan) [reply NEWS.md]: You can also `git checkout -- NEWS.md` and start over with changes to that file, which may be easier.

[the remark this answers is not in the document]
Comment (MasonProtter) [reply NEWS.md]: Thanks for the help. It appears to be fixed now. Hard to tell from the commit difs but if you look at https://github.com/JuliaLang/julia/pull/27447/files it appears kosher.

**Review by ararslan — APPROVED** [review]

File: NEWS.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ararslan) [reply NEWS.md]: Yep, looks good now. Nice work!

**Comment by StefanKarpinski** [comment]
This is such a slick little syntax that just falls out of the existing rules so nicely 💃

**Comment by MasonProtter** [comment]
I was a little skeptical of the `.` broadcast notation initially but now I think it’s just wonderful after realizing how extensible and compossible it is.

**Pull request merged**